app/src/main/res/drawable/splash_anim.xml

```diff
@@ -1,27 +1,6 @@
 <animated-vector xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:aapt="http://schemas.android.com/aapt">
-    <aapt:attr name="android:drawable">
-        <vector
-            android:width="64dp"
-            android:height="64dp"
-            android:tint="?attr/colorSecondary"
```

Comment:
I assume that this color has been intentionally removed for compatibility reasons?

With the default tint, the background contrast is quite bad on the dark theme with SDK >= 31.

Reply:
Yeah, that wasn't being used. Will revert that change and check again.

Reply:
I tried reverting the tint change, but the icon is always a black icon then, so I kept the change.

Reply:
I've added different `color` resources to make it look decent in light and in dark mode now, I think using the primary / secondary color doesn't make much sense here after rethinking it.